From 10caadd1a26a508d330c4f9c6f8eb966f42dbd22 Mon Sep 17 00:00:00 2001
From: Chris Packham <judge.packham@gmail.com>
Date: Thu, 28 Feb 2019 20:53:23 +1300
Subject: [PATCH] ARM: kirkwood: add spi0 alias for dreamplug

The conversion to DM_SPI managed to break accessing the environment on
dreamplug. This is because the environment code relies on being to able
to select the SPI device based on the sequence number. Add an alias so
that the spi0 bus gets sequence number 0.

Reported-by: Leigh Brown <leigh@solinno.co.uk>
Signed-off-by: Chris Packham <judge.packham@gmail.com>
---
 arch/arm/dts/kirkwood-dreamplug.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/dts/kirkwood-dreamplug.dts b/arch/arm/dts/kirkwood-dreamplug.dts
index a647a65c20..ccd74dd7fb 100644
--- a/arch/arm/dts/kirkwood-dreamplug.dts
+++ b/arch/arm/dts/kirkwood-dreamplug.dts
@@ -18,6 +18,10 @@
 		stdout-path = &uart0;
 	};
 
+	aliases {
+		spi0 = &spi0;
+	};
+
 	ocp@f1000000 {
 		pinctrl: pin-controller@10000 {
 			pmx_led_bluetooth: pmx-led-bluetooth {
-- 
2.20.1

